Content.Client/Blob/BlobbernautSystem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Linq;
+using Content.Client.DamageState;
+using Content.Shared.Blob;
+using Robust.Client.GameObjects;
+using Robust.Shared.GameStates;
+
+namespace Content.Client.Blob;
+
+public sealed class BlobbernautSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<BlobbernautComponent, ComponentHandleState>(OnBlobTileHandleState);
+    }
+
+    private void OnBlobTileHandleState(EntityUid uid, BlobbernautComponent component, ref ComponentHandleState args)
```

Comment:
This isn't a blob tile. Stop copy-pasting shit blindly, deduplicate your code. (x2)